src/app/components/game-components/kamisado/kamisado.component.ts

```diff
@@ -70,6 +70,9 @@ export class KamisadoComponent extends AbstractGameComponent<KamisadoMove, Kamis
         }
         if (this.chosen.x === -1) {
             return this.choosePiece(x, y);
+        } else if (x === this.chosen.x && y === this.chosen.y) {
+            // Do nothing, the user selected the already-selected piece
+            return MGPValidation.SUCCESS;
```

Comment:
Si à son premier tour il essaye de choisir une pièce, il sais pas revenir sur sa décision, rapy :p

Reply:
il sait pas revenir sur sa décision *en recliquant sur la pièce qu'il a déjà sélectionné*, mais bien en cliquant autre part, donc non-rapy et plutôt friendly je dirais !

Reply:
Dans l'user-friendlicité d'MGP faudrait prendre en compte l'homogénéité, donc si qqch marche dans un composant, qu'il marche partout.
Je sais que j'ai d'autres jeux où reclicker c'est désélectionner, j'vais demander par-ci par-là ce qui semble le plus instinctif des deux et on ticketera l'uniformisation de cet aspect de l'UI !
D'ailleurs on prendre les deux (sauf pour les jeux ou le dropping existe en plus de déplacement)